tests/test_samplers.py

```diff
@@ -116,7 +128,7 @@ class BernoulliNegativeSamplerTest(_NegativeSamplingTestCase, unittest.TestCase)
     def test_sample_bern(self):
         """Test if relations are not corrupted."""
         # Generate negative sample for additional tests
-        negative_batch = self.negative_sampler.sample(positive_batch=self.positive_batch)
+        negative_batch, _ = self.negative_sampler.sample(positive_batch=self.positive_batch)
```

Comment:
are there any tests you could apply to the mask rather than throwning it away?

Reply:
There are tests for the filter in lines 78:89

Reply:
Excellent, thanks for the clarification.